forest/src/daemon.rs

```diff
@@ -158,8 +158,10 @@ pub(super) async fn start(config: Config) {
 
     sync_from_snapshot(&config, &state_manager).await;
 
+    set_proofs_parameter_cache_dir_env(&config.data_dir);
```

Comment:
This was quite an unexpected behaviour that took some time to pinpoint. It would be nice to provide a helpful comment explaining why we actually need to do this.